Cognite.Simulator.Utils/FileLibrary.cs

```diff
@@ -215,6 +215,14 @@ public IEnumerable<Task> GetRunTasks(CancellationToken token)
             return new List<Task> { SaveStates(token), SearchAndDownloadFiles(token) };
         }
 
+        public void CreateDirectoryIfNotExists(string directoryPath)
```

Comment:
Consider making it private, if only using internal. Else add XML documentation

Reply:
Done